### ISSUE TITLE
Sign up / sign in

### ISSUE THREAD

**Issue by burmashave** [abevoelker/devise-passwordless#11]
Was trying out your gem - nice work! - just had a few questions. (Apologies if the answers are obvious.)

- when initially signing up, it seems any new email supplied to /users/sign_up/ is trusted and signed in, with a user record created for it. Subsequent sign ins (if logged out) require the magic link. Is there an easy way to require the token upon sign up too?

- is there any easy way to combine sign_in and sign_up views of Devise for a single form that accomplishes either/both when necessary?

**Comment by abevoelker** [comment]
No apologies necessary.

1. Hmm are you not using the `:confirmable` strategy? I think that should do what you want. If not let me know
2. Yeah you should be able to create a partial that both pages use. You can key off e.g. the `controller_name` to differentiate behavior. [Here's how Devise's shared links](https://github.com/heartcombo/devise/blob/main/app/views/devise/shared/_links.html.erb) partial works for an example.

**Comment by burmashave** [comment]
You're absolutely right: I had forgotten about `:confirmable`. Been a while since I've used Devise. Enabling it did the trick. Your note on the README about how to allow users to login directly from confirmable emails (without having to enter email a second time) was incredibly helpful, too.

And thanks for the tip about the multi-use partial. Keying off the controller name should do the trick.

I'll look into the Rails 7 errors, another set of eyes can't hurt.
